internal/fs/inode/dir.go

```diff
@@ -31,6 +31,11 @@ import (
 	"golang.org/x/net/context"
 )
 
+// ListObjects call supports fetching upto 5000 results via maxResults param in
+// one call. By default 1000 results are returned if value is not set.
```

Comment:
Can you also add that the 5000 is supportable only when the projection is not full

Reply:
done.